packages/spec_cli/test/utils.dart

```diff
@@ -85,7 +85,7 @@ class TestRenderer extends Renderer {
 
   @override
   void renderFrame(String output) {
-    final normalizedOutput = output.withoutAnsi.replaceFirst(
+    final normalizedOutput = output.strip.replaceFirst(
```

Comment:
It's kind of unclear what `String.strip` does at first glance. Not that it matters for this PR